packs/data/feats.db/lethal-finisher.json

```
@@ -10,7 +10,7 @@
             "value": 1
         },
         "description": {
-            "value": "<p>You stab your foe in a vital organ, possibly killing them outright. Make a Strike. On a success, you forego your precise strike damage from the finisher.</p>\n<p>Instead, your target takes additional precision damage based on a Fortitude save against your class DC. If your Strike was a critical hit, the target's saving throw outcome is one degree worse.</p>\n<hr />\n<p><strong>Critical Success</strong> You deal [[/r 6 #precision]]{6 precision damage}.</p>\n<p><strong>Success</strong> You deal [[/r {6d6}[precision]]]{6d6 precision damage}.</p>\n<p><strong>Failure</strong> You deal [[/r {12d6}[precision]]]{12d6 precision damage}.</p>\n<p><strong>Critical Failure</strong> You deal [[/r {18d6}[precision]]]{18d6 precision damage}.</p>"
+            "value": "<p>You stab your foe in a vital organ, possibly killing them outright. Make a Strike. On a success, you forego your precise strike damage from the finisher.</p>\n<p>Instead, your target takes additional precision damage based on a Fortitude save against your class DC. If your Strike was a critical hit, the target's saving throw outcome is one degree worse.</p>\n<hr />\n<p><strong>Critical Success</strong> You deal [[/r {6}[precision]]]{6 precision damage}.</p>\n<p><strong>Success</strong> You deal [[/r {6d6}[precision]]]{6d6 precision damage}.</p>\n<p><strong>Failure</strong> You deal [[/r {12d6}[precision]]]{12d6 precision damage}.</p>\n<p><strong>Critical Failure</strong> You deal [[/r {18d6}[precision]]]{18d6 precision damage}.</p>"
```

```suggestion
            "value": "<p>You stab your foe in a vital organ, possibly killing them outright. Make a Strike. On a success, you forego your precise strike damage from the finisher.</p>\n<p>Instead, your target takes additional precision damage based on a Fortitude save against your class DC. If your Strike was a critical hit, the target's saving throw outcome is one degree worse.</p>\n<hr />\n<p><strong>Critical Success</strong> You deal 6 precision damage.</p>\n<p><strong>Success</strong> You deal 6d6 precision damage}.</p>\n<p><strong>Failure</strong> You deal 12d6 precision damage.</p>\n<p><strong>Critical Failure</strong> You deal 18d6 precision damage.</p>"
```
We shouldn't have just precision rolls since precision needs a damage type to latch on to.

---

That's a good point. However, searching for [precision] shows 166 results. We should change them all?

---

Yeah, probably good to just remove them.